sunpy/roi/chaincode.py

```diff
@@ -139,7 +140,7 @@ def subBoundingBox(self, xedge=None, yedge=None):
             IndexMask = 1
             IndexValue = 0
         else:
-            print("Not edges input")
+            log.("Not edges input")
```

Comment:
`log.`?

Reply:
typo! thanks.

Reply:
This appears to have not actually been changed.